### PR TITLE
v8.5 what's new and upgrade guide

### PR DESCRIPTION
**Pull request by Pessimistress**
For #5948

### CODE REVIEW

File: docs/whats-new.md [hunk 1]
```diff
@@ -74,18 +61,47 @@ New props are added for more flexible styling of the texts:
...
+The layer also added support for automatically detecting the characters used in the data. Set `characterSet: 'auto'` to enable this feature.
 
 See [documentation](/docs/api-reference/layers/text-layer.md) for details.
 
-### React
 
-The `DeckGL` React component is rewritten using functional component and hooks.
+#### Tile3DLayer
+
+(TODO)
```
Comment (Pessimistress) [inline docs/whats-new.md:71]: @ibgreen

**Review by alasarr — APPROVED** [review]

**Review by kylebarron — APPROVED** [review]

File: docs/whats-new.md (hunk 1 above)
Comment (ibgreen) [reply docs/whats-new.md:71]: @belom88 @dryabinin-actionengine Let's suggest a good I3S dataset.

File: docs/whats-new.md [hunk 2]
```diff
@@ -74,18 +61,47 @@ New props are added for more flexible styling of the texts:
...
+- Tile3DLayer: multiple viewports (#5758)
+- Tile3DLayer: segmentation picking (#5757)
+- i3s: support vertex colors (#5756)
+- i3s: support uvRegions (#5760)
+- feat(geo-layers): Private MeshLayer with pbr material support (I3S) (#5761)
```
Comment (ibgreen) [inline docs/whats-new.md:76]:
```suggestion
- `Tile3DLayer` now takes full advantage of new features in I3S 1.7 tile sets, including
    - Page nodes (improved performance)
    - Draco compressed meshes (improved performance)
    - Compressed textures (improved performance)
    - PBR materials (#5761)
    - Vertex colors (#5756)
    - UVRegions (#5760)
```

**Review by ibgreen — APPROVED** [review]

File: docs/whats-new.md [hunk 3]
```diff
@@ -74,18 +63,50 @@ New props are added for more flexible styling of the texts:
...
+  + Draco compressed meshes (improved performance)
+  + Compressed textures (improved performance)
+  + PBR materials
+  + Vertex colors
+  + UVRegions
 
+
+#### Other layer improvements
+
+- `ScatterplotLayer` adds `billboard` mode
```
Comment (chrisgervang) [inline docs/whats-new.md:85]:
```suggestion
- `ScatterplotLayer` adds `billboard` mode
- `TripLayer` adds `fadeTrail` mode
```

**Pull request merged**